### ISSUE TITLE
Device profile switch entities gone

### ISSUE THREAD

**Issue by Marvilius** [mammuth/ha-fritzbox-tools#44]
Hey there

Und I used your addon for 2 weeks now without a trouble.
Yesterday the switch entities for the profiles were gone, while guest wifi was still there as well as port forward...whats goin on and how can i fix that?

Thx

**Comment by AaronDavidSchneider** [comment]
Thankyou for the report.
Could you provide more details: Logs, Fritzbox version, etc.

Could you also double check if the names of your devices Match the ones found in your configuration.

**Comment by Marvilius** [comment]
I never specified any devicenames prior to now. I use a fb 6890 lte with actual 7.10 version of fritz os

**Comment by AaronDavidSchneider** [comment]
Since the release of 1.5, one needs to specify the monitored devices.
Just re-add the integration and specify the devices:

```
fritzbox_tools:
  host: "192.168.178.1"  # required
  username: "home-assistant"  # required (create one at `System > FRITZ!Box Benutzer` on your router)
  password: "yourfritzboxpassword"  # required
  homeassistant_ip: "192.168.178.42"  # Optional. Needed if you want to control port forwardings for the device running Home Assistant
  profile_on: "Standard"  # Optional. Needed if you want to switch between device profiles ("Zugangsprofile")
  profile_off: "Gesperrt"  # Optional. Needed if you want to switch between device profiles ("Zugangsprofile")
  devices: # Optional. If you don't want to expose a profile switch for just some of your network devices
    - "Helens-iPhone"
    - "Aarons-MacBook-Air"
    - "..."
```

You might need to remove the integration manually (see `Configuration` - `Integration`) before updating configuration.yaml. You can configure Fritzbox Tools right there since 1.5.

**Comment by Marvilius** [comment]
Why is there a hashtag optional behind devices then if it is required?

My config looks like that:

fritzbox_tools:
host: 192.168.178.1 in upper quotes
username: in upper quotes
password: in upper quotes
homeassistant_ip: ip of my hassio in upper quotes

Did i miss anything?
Bg

Von meinem iPhone gesendet

**Comment by mammuth** [comment]
> Why is there a hashtag optional behind devices then if it is required?

Well, the setting is optional. You just need to explicitly name the devices you want to have profile switches for.
The reason for this is that the set up of the switches takes very long and most users only want to have switches for a couple of devices (if they use this feature at all).

Previously, we simply set up a switch for every profile.

I updated the readme to better reflect the new behavior.
